src/akeneo/api/helpers.ts

```diff
@@ -22,3 +28,65 @@ export function getHeaders(accessToken: string): Headers {
 export function getUrl(config: ApiConfig): string {
   return [config.pimUrl, config.endpoint].join("");
 }
+
+/**
+ * Builds a list of filters to apply to the list endpoint.
+ * @param filters The filters to apply.
+ * @returns The search query to apply.
+ */
+export function buildListFilters(
+  filters?: ListFilters | IdentifierFilter
+): string | undefined {
+  if (!filters) return undefined;
+
+  const result = [];
+
+  if ("uuids" in filters && filters.uuids) {
```

Comment:
Maybe we can make this simpler with `?.`?

```suggestion
  if (filters?.uuids?.length) {
```

Reply:
I tried that way :/ It really is the cleaner way to do it, but unfortunately, we need to first tell typescript that we're dealing with the `ListFilters` type, and only then can we access the field. Maybe I'm missing some typescript feature that could help us here.

![image](https://user-images.githubusercontent.com/29166076/201097831-f2e83e56-596a-4827-a5d0-f02126bd6c71.png)

Reply:
What do you suggest?

Reply:
Gotcha! No worries then - the current solution looks good!